compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+  proxy:
+    image: nginx
+    ports:
+      - "8080:80"
+    build: 
+      context: ./proxy
+    develop:
+      watch:
+        - path: ./proxy/www
+          action: sync
+          target: /data/www
```

Comment:
This feature was introduced in Docker Compose v2.22.0 but the version used on the cluster is v2.20.2:
![image](https://github.com/YetAnotherSpieskowcy/travel-agency-app/assets/6032823/ea971e94-7b7a-40ab-be4b-dd3a99a60d49)

Docker Compose does not ignore additional properties so it wouldn't work:
![image](https://github.com/YetAnotherSpieskowcy/travel-agency-app/assets/6032823/470de165-3a46-45ea-b383-b8e1732baaa0)

One potential solution would be to define an override file for development (defining a volume for the www dir) and then use something like `docker-compose -f compose.yaml -f compose.dev.yaml` for layering. There may be some other options I'm not familiar with that would work here as well. I feel like the override files might be helpful for different reasons in development as well though (binding `80` on production vs `8080` on dev, perhaps?) but it does mean having to manually specify the files.

Reply:
This doesn't matter since it's used for development purposes only.
This section can be stripped when development of frontend is mostly finished.
So unless we have a reason to test on cluster in the near future, I don't think any action should be taken in this case.